proto/substrait/algebra.proto

```diff
@@ -445,6 +447,8 @@ message Rel {
     HashJoinRel hash_join = 13;
     MergeJoinRel merge_join = 14;
     NestedLoopJoinRel nested_loop_join = 18;
+    DelimGetRel delim_get = 23;
```

Comment:
For clarity for the rest of the universe let's expand this out to DuplicateEliminatedGetRel (or at least DupElimGetRel).